test/upgrade_test/fission_objects.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set -e
+
+ns="f-ns"
+ROOT=$(pwd)
+REPO="docker.io/library"
+STABLE_VERSION=1.12.0
+HELM_VARS="helmVars=repository=docker.io/library,image=fission-bundle,pullPolicy=IfNotPresent,imageTag=latest,fetcher.image=docker.io/library/fetcher,fetcher.imageTag=latest,postInstallReportImage=reporter,preUpgradeChecksImage=preupgradechecks" 
+
+getVersion () {
+    echo $(git rev-parse HEAD)
+}
+
+getDate () {
+    echo $(date -u +'%Y-%m-%dT%H:%M:%SZ')
+}
+
+getGitCommit () {
+    echo $(git rev-parse HEAD)
+}
+
+getNameSpace () {
```

Comment:
Unused function.

Reply:
Removed this function since I added it for testing purpose.